### ISSUE TITLE
No data in sparklines/WebTSA

### ISSUE THREAD

**Issue by SRGDamia1** [ODM2/ODM2DataSharingPortal#193]
Starting March 5 at exactly midnight, a bunch of my data just disappears from WebTSA and the sparklines.  There is a non-zero value for the single most recent point, but that value seems to disappear.

For example, at site ROCK_US3:
![capture](https://user-images.githubusercontent.com/5374350/37367067-bd6dff8c-26d8-11e8-99ef-2cc253d00170.JPG)

![capture2](https://user-images.githubusercontent.com/5374350/37367072-c12a5d1e-26d8-11e8-8cf7-be770a6f4c99.JPG)

**Comment by jcaraballo17** [comment]
The data series database we're using started having problems with the data.
We were able to fix it and no data was lost, but there's still one minor issue related to this one. 

![image](https://user-images.githubusercontent.com/4700505/37804893-0d5cd4f6-2e0d-11e8-917f-9d7c4963be41.png)

The data table shows a null utc offset. We already issued a fix for this to be deployed along with the Hydroshare integration release in the next few days.
